README.md

```diff
@@ -95,3 +95,9 @@ With Vim, currently only PromQL queries inside YAML files work without additiona
 #### Debugging
 
 The Vim command `:YcmDebugInfo` gives status information and points to logfiles.
+
+### Sublime Text 3
+
+With ST3, you need to install package `LSP`, `LSP-promsql` via `Package Control`. Follow the installation instruction of [LSP-promql](https://github.com/nevill/lsp-promql#installation).
+
+Now, you can start editing any file with `.promql` as suffix.
```

Comment:
Do you plan to support `.yaml` files, too?

Reply:
Do you mean embeded promql in YAML? I don't know how to do that right now. The syntax highlight in ST3 can do, but not the LSP plugin. That's why I emphasized to edit a file with suffix`.promql`.

Reply:
You can simply enable the language server for all YAML files.

The language server then takes care of searching for PromQL queries inside that YAML file.

At least on VS Code this still works well when the YAML language server is enabled, too, and the performance overhead introduced by the PromQL language server is small compared to the resources taken by VS Code and the YAML language server.

Reply:
Now it works within YAML, as you can see [there](https://github.com/nevill/lsp-promql#usage).
It takes me some time until I realize that `promql-langserver` only works with specific nodes in YAML, basically in a Prometheus config YAML.